fpc.py

```diff
@@ -1069,7 +1069,7 @@ def findEndOfTemplate(text,template):
 support_templates = (u'[Ss]upport',u'[Pp]ro',u'[Ss]im',u'[Tt]ak',u'[Ss]í',u'[Pp]RO',u'[Ss]up',u'[Yy]es',u'[Oo]ui',u'[Kk]yllä', # First support + redirects
                      u'падтрымліваю',u'[Pp]our',u'[Tt]acaíocht',u'[Cc]oncordo',u'דעב',u'[Ww]eak support',
                      u'[Ss]amþykkt',u'支持',u'찬성',u'[Ss]for',u'за',u'[Ss]tödjer',u'เห็นด้วย',u'[Dd]estek',
-                     u'[Aa] favore?',u'[Ss]trong support',u'[Ss]Support', u'Υπέρ', u'[Ww]Support', u'[Ss]' )
+                     u'[Aa] favore?',u'[Ss]trong support',u'[Ss]Support', u'Υπέρ', u'[Ww]Support', u'[Ss]', u'[Aa]poio' )
```

Comment:
This change was not included originally when I merged, have you intentionally skipped this or should it also be included?

Reply:
I added the template